src/test/java/steps/Meetings.java

```diff
@@ -97,7 +98,7 @@ public void theSecondMeetingInformationShouldNotBeDisplayedInTheNextSection(){
 
     @And("^the second Meeting should not be listed in the Meetings of Room using the API$")
     public void theSecondMeetingShouldNotBeListedInTheMeetingsOfRoomUsingTheAPI(){
-        assertTrue(true);
+        assertFalse(apiMeetingMethods.isMeetingPresent(meetingEntity2.getSubject(),"title", mainTabletPage.getMainTitle()));
```

Comment:
at this point you should send the entity to apiMeetingMethods like this
apiMeetingMethods.isMeetingPresent(meetingEntity)